person.rb

```diff
@@ -19,3 +22,7 @@ def of_age?
     @age >= 18
   end
 end
+
+def correct_name
+  @name
+end
```

Comment:
- [ ] Kindly move this function `correct_name` inside the person class as it's now not part of the class.